src/py_eddy_tracker/dataset/grid.py

```diff
@@ -2264,6 +2264,16 @@ def from_netcdf_list(cls, filenames, t, x_name, y_name, indexs=None, heigth=None
             new.datasets.append((t, d))
         return new
 
+    def shift_files(self, t, filename, x_name, y_name, indexs, heigth):
```

Comment:
Maybe add default value to index and height? And maybe kwargs directly to regular grid

Reply:
ok for default values, but what do you mean by kwargs directly to regular grid ?

Reply:
Lot of those arguments are not used by method just transfer to another one, so maybe we could **kwargs to be transparent

Reply:
ok, done